package.json

```diff
@@ -41,6 +41,7 @@
 		"@tanstack/query-core": "^4.36.1",
 		"@tanstack/query-persist-client-core": "^4.36.1",
 		"@tanstack/query-sync-storage-persister": "^4.36.1",
-		"@tanstack/svelte-query": "^4.36.1"
+		"@tanstack/svelte-query": "^4.36.1",
+		"datetrigger": "^1.0.0"
```

Comment:
wouldn't be a bad idea to bump this (and to update the pnpm lock if you can)

Reply:
idk what's wrong but I can't get it to work no matter what using pnpm

![FKphwp8.png](https://github.com/ReVanced/revanced-website/assets/100418457/212b9ab9-3d73-4711-89e2-267eea1f5f0c)

Reply:
`rm -rf node_modules` and then try again

Reply:
did that multiple times, nothing changed, same error

Reply:
@KTibow https://streamable.com/550jyf